openc3-init/plugins/packages/openc3-tool-tlmviewer/src/tools/TlmViewer/Openc3Screen.vue

```diff
@@ -409,17 +419,15 @@ export default {
       this.screenKey = Math.floor(Math.random() * 1000000)
       // After re-render wait and see if there are errors before saving
       this.$nextTick(function () {
-        if (this.errors.length === 0) {
```

Comment:
Sometimes I was getting errors from I think TEMP2 due to the nil / undefined return values. This prevented saving the screen. I think it's better to let them save the screen no matter what and the error messages still identify what's going on.